### ISSUE TITLE
UI command returns "No codebase UI configured" Error

### ISSUE THREAD

**Issue by bytesource** [unisonweb/unison#4085]
Hi,

When typing `ui` in `ucm`, a browser tab opens at http://127.0.0.1:49919/wyPtF58cUaT_Hxc-/ui, displaying the following text:

> No codebase UI configured. Set the UCM_WEB_UI environment variable to the directory where the UI is installed.

All other `ucm` commands that I have tried so far are working.

Setup:
-- Windows 10 
-- VS Code 1.78.2
-- Unison VS Code Plugin v0.08
-- Browser: Firefox, Chrome

Did I make a mistake opening the local codebase UI?

**Comment by aryairani** [comment]
Hi @bytesource, which version of `ucm` are you running?

If you built your own binary, you need to run `dev-ui-install.sh` or perform the windows equivalent of its contents to get the html files for the UI command.

**Comment by aryairani** [comment]
Going to close this for now, but feel free to reopen if you weren't using a "release" version of UCM, and running that script doesn't fix the issue.

**Comment by bytesource** [comment]
Sorry, I missed your reply. 

I'm running UCM version M4i. 

I ran the script which I found here: https://github.com/unisonweb/unison/blob/trunk/dev-ui-install.sh

and I ended up with the file `unisonLocal.zip` in my root folder. It appears that `ui` folder creation and unpacking didn't work, although I ran the script as an administrator. 

![image](https://github.com/unisonweb/unison/assets/610307/b1f6103d-d1f9-4e43-85b7-2137e968c0dc)
